Changes to language tag

Nutomic opened the pull request:
- Dont use bright white color because its not that important
- Hide tag in case of undetermined (doesnt add any information)

Before:
![Screenshot_20230523_221859](https://github.com/LemmyNet/lemmy-ui/assets/1044450/aab0fd42-415b-4fb0-bba1-376834771bbc)

After:
![Screenshot_20230523_221840](https://github.com/LemmyNet/lemmy-ui/assets/1044450/05571c6d-9778-4420-8abc-1fa099967a02)

Not sure about the specific css classes, we can also change it as long as it's gray like the other text and not bright white.

SleeplessOne1917 commented on src/shared/components/comment/comment-node.tsx:
```diff
@@ -283,13 +283,15 @@ export class CommentNode extends Component<CommentNodeProps, CommentNodeState> {
-                  )?.name
-                }
-              </span>
+              {cv.comment.language_id != 0 && (
```
Nitpick: !== instead of !=.

SleeplessOne1917 commented on src/shared/components/post/post-listing.tsx:
```diff
@@ -344,13 +344,15 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
-            )?.name
-          }
-        </span>
+        {post_view.post.language_id != 0 && (
```
Same as other comment.

SleeplessOne1917 commented:
Regarding the css class badges: you can get a badge of a given color with "badge badge-(color name here)". The color name would be something from the theme, like light, dark, info, primary etc. Text gets colored similarly with class names like "text-(theme color)".

Also, how does this look in lightly?

Nutomic commented:
Good suggestion, `badge-muted` looks decent as its the same color used for other text in that area (like the creation time). I also changed the admin/mod badges to muted (see https://github.com/LemmyNet/lemmy-ui/issues/1037) and moved the creator badge first.

![Screenshot_20230524_122115](https://github.com/LemmyNet/lemmy-ui/assets/1044450/1551ef0a-0ceb-4820-8656-bc1df9d8ee6a)

![Screenshot_20230524_122133](https://github.com/LemmyNet/lemmy-ui/assets/1044450/0fcd8cb6-4cf0-4c8f-a53f-fa0d8ca86857)

Nutomic commented on src/shared/components/comment/comment-node.tsx:
```diff
@@ -240,23 +240,23 @@ export class CommentNode extends Component<CommentNodeProps, CommentNodeState> {
               )}
               {isAdmin_ && (
-                <div className="badge badge-light d-none d-sm-inline mr-2">
+                <div className="badge badge-muted d-none d-sm-inline mr-2">
```
Youre right it actually has no effect. So I removed `badge-muted`, only leaving `badge` and it still looks the same as in the screenshots.

dessalines commented on src/shared/components/post/post-listing.tsx:
```diff
@@ -345,7 +345,7 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
         </li>
         {post_view.post.language_id !== 0 && (
-          <span className="mx-1 badge badge-muted">
+          <span className="mx-1 badge">
```
You're not supposed to use it without the 2nd `badge-x` class, as it might look weird on some themes. Did you try `badge-secondary` ?

SleeplessOne1917 replied:
Secondary will be orange on lightly I think. It'll bring us back to the badge being too prominent.

Nutomic replied:
Well its working...

dessalines replied:
If it looks good on both the litely and darkly, I have no problems merging. Might give issues on other themes tho.
